docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -1,238 +1,200 @@
 
-*Solution*:
+[float]
+[[alerting-index-threshold-chart]]
+=== Preview the index threshold rule chart
 
-Rules and actions run as background tasks by each {kib} instance at a default rate of ten tasks every three seconds.
+When creating or editing an index threshold rule, you see a graph of the data the rule will operate against, from some date in the past until now, updated every 5 seconds.  
+[role="screenshot"]
+image::images/rule-details-alerts-inactive.png[Index Threshold chart]
 
-If many rules or actions are scheduled to run at the same time, pending tasks will queue in {es}. Each {kib} instance then polls for pending tasks at a rate of up to ten tasks at a time, at three second intervals. Because rules and actions are backed by tasks, it is possible for pending tasks in the queue to exceed this capacity and run late.
+The end date is related to the alert interval (IIRC, 30 “intervals” worth of time). You can use this view to see if the rule is getting the data you expect, and visually compare to the threshold value (a horizontal line in the graph). If the graph does not contain any lines except for the threshold line, then the rule has an issue, for example, no data is available given the specified index and fields or there is a permission error.
+Diagnosing these may be difficult - but there may be log messages for error conditions. 
 
-For details on diagnosing the underlying causes of such delays, see <<task-manager-health-tasks-run-late>>.
+[float]
+[[alerting-rest-api]]
+=== Use the REST APIs
 
-Alerting and action tasks are identified by their type.
+There is a rich set of HTTP endpoints to introspect and manage rules and connectors.
+One of the http endpoints available for actions is the POST _execute API (reference).  You can use this to “test” an action.  For instance, if you have a server log action created, you can execute it via curling the endpoint:
+[source, txt]
+--------------------------------------------------
+curl -X POST -k \
+ -H 'kbn-xsrf: foo' \
+ -H 'content-type: application/json' \
+ https://elastic:changeme@localhost:5601/api/actions/connector/a692dc89-15b9-4a3c-9e47-9fb6872e49ce/_execute \
+ -d '{"params":{"subject":"hallo","message":"hallo!","to":["me@example.com"]}}'
+--------------------------------------------------
 
-* Alerting tasks always begin with `alerting:`. For example, the `alerting:.index-threshold` tasks back the <<rule-type-index-threshold, index threshold stack rule>>.
-* Action tasks always begin with `actions:`. For example, the `actions:.index` tasks back the <<index-action-type, index action>>.
+In addition, there is a command-line client that uses legacy Rules and Connectors APIs, which can be easier to use, but must be updated for the new APIs.
+CLI tools to list create, edit, and delete alerts and actions are available in https://github.com/pmuellr/kbn-action[kbn-action], which you can install as follows:
```

Comment:
I don't think we should link to `kbn-action` in our docs; especially since the cross-version support is kinda iffy - you can use some older versions of the tool with some older versions of Kibana, but mostly, older versions of Kibana will not work with newer versions of the tool, and vice versa.